CellMigStandOrg/biotracks
# Unify standard names

Addresses #42.

To test:

 * check that the Travis build stays green
 * check that occurrences of old names have been replaced by the corresponding new ones

Changes summary:

 * All of our standard field names now have a `cmso` **prefix**, as discussed in the meetings
 * `frame` changes to `frame_id` since it's an ID just like `{objects,links,tracks}_id`
